torch_em/data/datasets/light_microscopy/cvz_fluo.py

```diff
@@ -0,0 +1,167 @@
+"""The CVZ-Fluo dataset contains annotations for cell and nuclei segmentation in
+fluorescence microscopy images.
+
+NOTE: You need to install 'synapseclient`, create an account and authentication token on Synapse,
+and follow the documentation to create your authentication config: https://python-docs.synapse.org/tutorials/authentication/
+and install using 'synapse get -r syn27624812'.
+
+The dataset is from the publication https://doi.org/10.1038/s41597-023-02108-z.
+Please cite it if you use this dataset for your research.
+"""  # noqa
+
+import os
+from glob import glob
+from tqdm import tqdm
+from pathlib import Path
+from natsort import natsorted
+from typing import Union, Literal, Tuple, Optional, List
+
+import imageio.v3 as imageio
+from skimage.measure import label as connected_components
+
+from torch.utils.data import Dataset, DataLoader
+
+import torch_em
+
+from .. import util
+from .neurips_cell_seg import to_rgb
```

Comment:
How many channels does the data have and how exactly do we deal with it?

Reply:
The ROIs for cell images have all 3-channels. However for ROIs with DAPI-staining, some images have 1 channel and others 3, which leads to collating issues.